### PR TITLE
Feature apply liniting checks

### PR DESCRIPTION
**Pull request by Trishisingh**

### CODE REVIEW

File: de_workloads/ingest/Ingest_AzureSql_Example/de-ingest-azuresql-example.yml [hunk 1]
```diff
@@ -75,6 +75,10 @@ stages:
           - template: ../../../build/azDevOps/templates/air-infrastructure-data-setup.yml
             parameters:
               TaskctlVersion: ${{ variables.TaskctlVersion }}
+          - script: pip install --upgrade pre-commit
```
Comment (motylp) [inline de_workloads/ingest/Ingest_AzureSql_Example/de-ingest-azuresql-example.yml:78]: Any changes like that should be also introduced to jinja templates.

File: de_templates/ingest/Ingest_SourceType_SourceName/de-ingest-ado-pipeline.yml.jinja [hunk 2]
```diff
@@ -76,6 +76,12 @@ stages:
           - template: ../../../build/azDevOps/templates/air-infrastructure-data-setup.yml
             parameters:
               TaskctlVersion: {% raw %}${{ variables.TaskctlVersion }}{% endraw %}
+          - script: pip install --upgrade pre-commit tomli
```
Comment (balpurewal) [inline de_templates/ingest/Ingest_SourceType_SourceName/de-ingest-ado-pipeline.yml.jinja:79]: do you think we should have a taskctl task instead that installs pre-commit and also runs it and then call that from the pipeline so we are not repeating the code?

Comment (Trishisingh) [reply de_templates/ingest/Ingest_SourceType_SourceName/de-ingest-ado-pipeline.yml.jinja:79]: @balpurewal For now it's fine here as we already have linting standards in taskctl for wider usage These pre-commit checks are just for data which include Python code etc

**Review by satenderrathee — APPROVED** [review]

**Pull request merged**